adapters/smilewanted/smilewanted_test.go

```diff
@@ -0,0 +1,20 @@
+package smilewanted
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderSmileWanted, config.Adapter{
+		Endpoint: "http://prebid-server.smilewanted.com"})
```

Comment:
Nitpick: Please consider using an obviously fake domain for the tests. There is no need to use a real address here.